cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -275,6 +275,15 @@ impl pallet_assets::Config<TrustBackedAssetsInstance> for Runtime {
 	type BenchmarkHelper = ();
 }
 
+// Allow Freezes for the `Assets` pallet
+pub type TrustBackedAssetsFreezerInstance = pallet_assets_freezer::Instance1;
+impl pallet_assets_freezer::Config<TrustBackedAssetsFreezerInstance> for Runtime {
+	type FreezeIdentifier = ();
```

Comment:
Don't we need to set this?
```suggestion
	type FreezeIdentifier = RuntimeFreezeReason;
```